components/HeaderedControls/samples/HeaderedTreeViewSample.xaml

```diff
@@ -0,0 +1,36 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="HeaderedItemsControlExperiment.Samples.HeaderedTreeViewSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:CommunityToolkit.WinUI.Controls"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:HeaderedItemsControlExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Grid>
+        <controls:HeaderedTreeView Footer="Footer"
+                                   Header="Header"
+                                   ItemsSource="{x:Bind Items, Mode=OneWay}">
+            <controls:HeaderedTreeView.HeaderTemplate>
+                <DataTemplate>
+                    <TextBlock FontWeight="SemiBold"
+                               Text="{Binding}" />
+                </DataTemplate>
+            </controls:HeaderedTreeView.HeaderTemplate>
+            <controls:HeaderedTreeView.FooterTemplate>
+                <DataTemplate>
+                    <TextBlock FontWeight="SemiBold"
+                               Text="{Binding}" />
+                </DataTemplate>
+            </controls:HeaderedTreeView.FooterTemplate>
+            <controls:HeaderedTreeView.ItemTemplate>
+                <DataTemplate x:DataType="local:ExplorerItem">
+                    <TreeViewItem Content="{x:Bind Name}"
+                                  IsExpanded="True"
+                                  ItemsSource="{x:Bind Children}" />
```

Comment:
```suggestion
                    <muxc:TreeViewItem Content="{x:Bind Name}"
                                  IsExpanded="True"
                                  ItemsSource="{x:Bind Children}" />
```

This by default is a WUX TreeViewItem in UWP, which doesn't jive... needs to be MUXC TreeViewItem in a MUXC TreeView, missing namespace inclusion at the top should have been a giveaway too 😋